docs/Helm.md

```diff
@@ -0,0 +1,106 @@
+```
+
+To upgrade the operator, specify override configurations through the CLI or a new file:
+
+```
+helm upgrade --set splunkOperator.clusterWideAccess=false <RELEASE_NAME> splunk/splunk-operator -n <RELEASE_NAMESPACE>
+```
+
+Here we've upgraded the operator by revoking cluster-wide access.
+
+To rollback the operator to a previous release:
+
+```
+helm rollback <RELEASE_NAME> <REVISION_NUMBER> -n <RELEASE_NAMESPACE>
+```
+
+To clean-up resources and uninstall the operator:
+
+```
+helm uninstall <RELEASE_NAME> -n <RELEASE_NAMESPACE>
+```
+
+### Configuring Splunk Operator deployments
+
+There are a couple ways you can configure your operator deployment
+
+1. Using a ```new_values.yaml``` file to override default values (Recommended)
+```
+helm install -f new_values.yaml <RELEASE_NAME> splunk/splunk-operator -n <RELEASE_NAMESPACE>
+```
+
+2. Using the Helm CLI to set new values
+```
+helm install --set <KEY>=<VALUE> <RELEASE_NAME> splunk/splunk-operator -n <RELEASE_NAMESPACE>
+```
+
+Read more about configuring values [here](https://helm.sh/docs/intro/using_helm/).
+
+## Splunk Enterprise deployments
+
+The Splunk Enterprise chart allows you to install and configure Splunk Enterprise custom resources.
+
+First build the Splunk Operator chart as a dependency:
+```
+helm dependency build splunk/splunk-enterprise
+```
+
+To install a configured Splunk Enterprise deployment:
+```
+helm install <RELEASE_NAME> splunk/splunk-enterprise -n <RELEASE_NAMESPACE>
+```
+
+This chart installs the Splunk Operator as a dependency by default.
+
+If the operator is already installed then you will need to disable the dependency:
+```
+helm install --set splunk-operator.enabled=false <RELEASE_NAME> splunk/splunk-enterprise -n <RELEASE_NAMESPACE>
+```
+
+To see all configurable values contained in the ```values.yaml``` file:
+```
+helm show values splunk/splunk-enterprise
+```
+## Splunk Validated Architecture deployments
+
+The Splunk Enterprise chart has support for three Splunk Validated Architectures:
+
+- [Single Server Deployment (S1)](https://www.splunk.com/pdfs/technical-briefs/splunk-validated-architectures.pdf#page=9)
+- [Distributed Clustered Deployment + SHC - Single Site (C3)](https://www.splunk.com/pdfs/technical-briefs/splunk-validated-architectures.pdf#page=14)
+- [Distributed Clustered Deployment + SHC - Multi-Site (M4)](https://www.splunk.com/pdfs/technical-briefs/splunk-validated-architectures.pdf#page=20)
+
+Install a Standalone deployment using the following command:
+```
+helm install --set s1.enabled=true <RELEASE_NAME> splunk/splunk-enterprise -n <RELEASE_NAMESPACE>
+```
+Visit the Splunk Operator github repository to learn more about the configurable values of [splunk/splunk-operator](https://github.com/splunk/splunk-operator/blob/develop/helm-chart/splunk-operator/values.yaml) and [splunk/splunk-enterprise](https://github.com/splunk/splunk-operator/blob/develop/helm-chart/splunk-enterprise/values.yaml).
+
+
+
+
+
```

Comment:
Yes, that is in the pipeline. In the future, we are hoping to migrate existing deployments into helm releases. This may be possible by annotating all existing resources with the following fields:
`meta.helm.sh/release-name=<RELEASE_NAME>`
`meta.helm.sh/release-namespace=<RELEASE_NAMESPACE>`
`app.kubernetes.io/managed-by=Helm`
However, we need to explore this further. At the moment, the only solution is to create an entirely new release.

Reply:
Got it. If not in this PR, let us give that clarity in the documentation, so that we can avoid confusion.